packages/squiggle-lang/src/rescript/Distributions/PointSetDist/Continuous.res

```diff
@@ -280,6 +280,40 @@ let combineAlgebraicallyWithDiscrete = (
   }
 }
 
+let combineAlgebraicallyWithDiscreteFirst = (
+  op: Operation.algebraicOperation,
+  t1: PointSetTypes.discreteShape,
+  t2: t,
+) => {
+  let t1s = t1.xyShape
+  let t2s = t2->getShape
+
+  if XYShape.T.isEmpty(t1s) || XYShape.T.isEmpty(t2s) {
+    empty
+  } else {
+    let continuousAsLinear = switch t2.interpolation {
+    | #Linear => t2
+    | #Stepwise => stepwiseToLinear(t2)
+    }
+
+    let combinedShape = AlgebraicShapeCombination.combineShapesDiscreteContinuous(
+      op,
+      t1s,
+      continuousAsLinear |> getShape,
```

Comment:
Our older functions were written with the key argument last, not first. I don't recommend reordering outer calls to just use ``->``, that seems like it would make things more confusing. 

Eventually we can refactor the functions themselves, and that way we'd get the right order call.